src/transformers/generation_utils.py

```diff
@@ -1139,7 +1139,7 @@ def generate(
         >>> sentence = "Paris is one of the densest populated areas in Europe."
         >>> input_ids = tokenizer(sentence, return_tensors="pt").input_ids
 
-        >>> outputs = model.generate(input_ids)
+        >>> outputs = model.generate(input_ids, num_beams=5, early_stopping=True)
```

Comment:
```suggestion
        >>> outputs = model.generate(input_ids, num_beams=5)
```
Let's showcase beam search only.